lib/AppConfig.php

```diff
@@ -140,10 +163,28 @@ public function getAppValue($key) {
 	 * @param string $key
 	 * @param string $value
 	 *
-	 * @return string
+	 * @return void
 	 */
 	public function setAppValue($key, $value) {
-		return $this->config->setAppValue($this->appName, $key, $value);
+		$this->validateValue($key, $value);
+		$this->config->setAppValue($this->appName, $key, $value);
+	}
+
+	/**
+	 * @param string $key
+	 * @param string $value
+	 *
+	 * @return void
+	 */
+	public function validateValue($key, $value) {
```

Comment:
Any reason for this method to be public? If so, it should be documented that the method can throw an "UnexpectedValueException" under some conditions

Reply:
`throws` has been added. As it was called Initially  from a different class I decided do not modify the visibility  when I moved the usage into the same class.